### ISSUE TITLE
question about the model size,why the size of model i trained is 240m,but the pretrained model is only 93m?

### ISSUE THREAD

**Issue by zfs1993** [davidsandberg/facenet#886]
i follow the guide 
https://github.com/davidsandberg/facenet/wiki/Classifier-training-of-inception-resnet-v1

nohup /opt/app/anaconda3/bin/python -u src/train_softmax.py --logs_base_dir ./logs/facenet/ --models_base_dir /opt/app/OCR/face_mobile/facenet-master/models/facenet --data_dir /opt/app/OCR/face-for-test/celeb_clean_align_mtcnn_182 --image_size 160 --batch_size 90 --model_def models.inception_resnet_v1 --lfw_dir /opt/app/OCR/face-for-test/lfw_mtcnnpy_160 --lfw_pairs /opt/app/OCR/face_mobile/facenet-master/data/pairs.txt --optimizer ADAM --learning_rate -1 --max_nrof_epochs 500 --keep_probability 0.8 --random_crop --random_flip  --use_fixed_image_standardization  --learning_rate_schedule_file data/learning_rate_schedule_classifier_msceleb_new.txt --weight_decay 5e-4  --embedding_size 512 --lfw_distance_metric 1 --lfw_use_flipped_images --lfw_subtract_mean --validation_set_split_ratio 0.01 --validate_every_n_epochs 5 --prelogits_norm_loss_factor 5e-4  2>&1 > run_softmax_v2_20180419.log &

why my model is much bigger than the model author given?
by the way, i want to apply the model to phone ,it is still too big after i quantized it , is anyone who has advice about a new model?

**Comment by AlexeyAL1e** [comment]
what accuracy did you get using these parameters?
